### PR TITLE
done with simple testing env, asynchronous error with testing need to fix

### PR DESCRIPTION
**Pull request by eggtart26**

### CODE REVIEW

File: client/dist/style.css [hunk 1]
```diff
@@ -71,3 +71,13 @@ ul,ol {
     clear: both;
     display: table;
   }
+
+  ._26piifo {
+    overflow-wrap: break-word !important;
```
Comment (groverpankaj) [inline client/dist/style.css]: Don't name your class names as '._26iifo'. Use meaningful names

File: client/src/components/Gallery.jsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+                    <li><a href='#'>Help</a></li>
+                </ul>
+                <div
+                    style={{
+                        width: '200px',
+                        height: '80px',
+                        backgroundColor: 'yellow',
+                        fontSize: '24px',
+                        textAlign: 'center'
+                    }}
```
Comment (groverpankaj) [inline client/src/components/Gallery.jsx]: Move this style as class to css file, avoid using CSS inline style

File: client/src/components/Gallery.jsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+                <h1>From Local DataBase</h1>
+                <p>                    </p>
+                <h2 style={{
+                    overflow_wrap: 'break_word',
+                    font_family: 'Circular',
+                    font_size: '24px',
+                    font_weight: '800',
+                    line_height: '1.25em',
+                    color: 'rgb(72, 72, 72)',
+                    margin: '0px',
```
Comment (groverpankaj) [inline client/src/components/Gallery.jsx]: avoid inline style

File: client/src/components/Gallery.jsx (hunk 2 above)
Comment (eggtart26) [reply client/src/components/Gallery.jsx]: got it, it is for testing, i will clear all of it in later version

**Pull request merged**